README.md

```diff
@@ -109,40 +109,40 @@ the cockpit server. See more details [in the documentation](
 web/README.md#using-a-development-server).
 
 * Start the services:
-    * beware that D-Installer must run as root (like YaST does) to do
+    * beware that Agama must run as root (like YaST does) to do
       hardware probing, partition the disks, install the software and so on.
     * Note that `setup.sh` sets up D-Bus activation so starting manually is
       only needed when you prefer to see the log output upfront.
 
 ```console
 $ cd service
-$ sudo bundle exec bin/d-installer
+$ sudo bundle exec bin/agama
 ```
 
-* Check that D-Installer services are working with a tool like
+* Check that Agama services are working with a tool like
 [busctl](https://www.freedesktop.org/wiki/Software/dbus/) or
 [D-Feet](https://wiki.gnome.org/Apps/DFeet) if you prefer a graphical one:
 
 
 ```console
-$ busctl --address=unix:path=/run/d-installer/bus \
+$ busctl --address=unix:path=/run/agama/bus \
     call \
-    org.opensuse.DInstaller \
-   /org/opensuse/DInstaller/Manager1 \
-    org.opensuse.DInstaller.Manager1 \
+    org.opensuse.Agama1 \
+   /org/opensuse/Agama1/Manager \
+    org.opensuse.Agama1.Manager \
     CanInstall
 
-$ busctl --address=unix:path=/run/d-installer/bus \
+$ busctl --address=unix:path=/run/agama/bus \
     call \
-    org.opensuse.DInstaller.Language \
-   /org/opensuse/DInstaller/Language1 \
+    org.opensuse.Agama.Language1 \
+   /org/opensuse/Agama/Language1 \
     org.freedesktop.DBus.Properties \
-    GetAll s org.opensuse.DInstaller.Language1
+    GetAll s org.opensuse.Agama.Language1
 ```
 
 ## How to Contribute
 
-If you want to contribute to D-Installer, then please open a pull request or report an issue. You can also have a look to our [road-map](https://github.com/orgs/yast/projects/1/views/1).
+If you want to contribute to Agama, then please open a pull request or report an issue. You can also have a look to our [road-map](https://github.com/orgs/yast/projects/1/views/1).
```

Comment:
What about the GH Project in the `yast` org? We can keep it now but is that the plan for the future?

Reply:
I do not have a plan yet. I would like to move it, but I am not sure whether it is worth. After all, it gets outdated rather often.